common/src/main/scala/uk/ac/wellcome/message/MessageReader.scala

```diff
@@ -0,0 +1,32 @@
+package uk.ac.wellcome.message
+
+import com.amazonaws.services.sqs.model.{Message => SQSMessage}
```

Comment:
Why do we rename the import? It's a bit confusing cause we have another `SQSMessage` class